moonstone/filtering/mean_filtering.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, dataframe, threshold: float = None, percentage_to_keep: Optio
                                    overridden if threshold is set
         """
         super().__init__(dataframe)
+        self.raw_reads_number = self.df.sum().sum()
```

Comment:
Isn't it already defined due to his inheritance of CountsFiltering?

Reply:
indeed, I think I have changed that before the merging and it might have came back with the conflicts between the two branches, thanks for catching that!